starter_code/src/movies.js

```diff
@@ -1,20 +1,88 @@
 /* eslint no-restricted-globals: 'off' */
-// Turn duration of the movies from hours to minutes 
-
-
+var movies = require("./data.js");
+// Turn duration of the movies from hours to minutes
+function convertHoursToMinutes(hours){
+  var minutesFromHours = hours.split("h");
+  return minutesFromHours[0]*60;
+}
+function convertMinutesToMinutes(minutes){
+  var minutesFromMinutes = minutes.split("min");
+  return Number(minutesFromMinutes[0]);
+}
+function doTheMath(duration){
+  var timePieces = duration.split(" ");
+  var minutes = duration.reduce(function(sum, piece){
+    if(piece.includes("h")){
+      return sum + convertHoursToMinutes(piece)
+    }
+    return sum + convertMinutesToMinutes(piece)
+  },0);
+  return minutes;
+}
+function turnHoursToMinutes(movies){
+  var newMoviesArray = movies.map(function(movie){
+    var newMovie = {};
+    newMovie.title = movie.title;
+    newMovie.year = movie.year;
+    newMovie.director = movie.director;
+    newMovie.duration = doTheMath(movie.duration);
+    newMovie.genre = movie.genre;
+    newMovie.rate = movie.rate;
+    return newMovie;
+  })
+  return newMoviesArray;
+}
 // Get the average of all rates with 2 decimals 
-
-
+function ratesAverage(movies){
+  var theSum = movies.reduce(function(sum, movie){
+    return sum + Number(movie.rate);
+  },0);
+  var rateAvg = theSum / movies.length;
+  return Math.round(rate.avg*100) / 100;
+}
 // Get the average of Drama Movies
-
-
+function dramasOnly(movies){
+  var dramas = movies.filter(function(movie){
+      return movie.genre,includes("Drama")
+  })
+  return dramas;
+}
+function dramaMovieRate(movies){
+  var dramas = dramasOnly(movies);
+  If (dramas.length === 0){
+    return;
```

Comment:
You would not add "empty" `returns`, because they don't return anything, so they would serve no purpose. Perhaps you could have written this function differently to avoid having to do that or you could have used a `break` rather then a `return`.